manifests/ppa.pp

```diff
@@ -6,6 +6,8 @@
   $package_name   = $::apt::ppa_package,
   $package_manage = false,
 ) {
+  Apt::Ppa[$title] -> Class['apt::update']
```

Comment:
Does this work/make sense, to have in the same resource declaration? Also, since the necessary resources already notify `apt::update`, isn't that enough?